toucan_connectors/common.py

```diff
@@ -23,12 +23,17 @@
 RE_GET_KEEP_TYPE = r'{{(__keep_type__[^({{)}]*)}}'
 
 
+class NonValidEndpointVariable(Exception):
+    """ Error thrown for a non valid variable in endpoint """
+
+
 def nosql_apply_parameters_to_query(query, parameters):
     """
     WARNING : DO NOT USE THIS WITH VARIANTS OF SQL
     Instead use your client library parameter substitution method.
     https://www.owasp.org/index.php/Query_Parameterization_Cheat_Sheet
     """
+    print('first query ', query)
```

Comment:
Have you forgotten to remove this `print`? :)

Reply:
Oops! :blush: